test/test-framework/deploy/olm-catalog/memcached-operator/0.0.3/memcached-operator.v0.0.3.clusterserviceversion.yaml

```diff
@@ -0,0 +1,165 @@
+apiVersion: operators.coreos.com/v1alpha1
```

Comment:
Do we need a new CSV version?

Reply:
Not really. I felt like it made sense though since I added a new CRD to the operator